### PR TITLE
refactor: integration and separation

### PR DESCRIPTION
**Pull request by gyuguen**
### Implement
* Refactorying
  * broadcast function
  * Change name `TxBuilder` to `TxClient`
  * Added `TxClient` in `Service`
  * Moved `service/handler_data_validation` to ` server/service/datadeal/handler_data_validation`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,6 +17,12 @@ build: go.sum
 test:
 	$(GO) test -v ./...
 
+lint:
+	GO111MODULE=off go get github.com/golangci/golangci-lint/cmd/golangci-lint
+	golangci-lint run --timeout 5m0s --allow-parallel-runners
+	find . -name '*.go' -type f -not -path "./vendor*" -not -path "*.git*" | xargs gofmt -d -s
+	go mod verify
```
Comment (youngjoon-lee) [inline Makefile:24]: This won't used by CI, but for running this on your local?

**Review by youngjoon-lee — APPROVED** [review]
lgtm

File: Makefile (hunk 1 above)
Comment (gyuguen) [reply Makefile:24]: Yes, It used only local.

**Review by audtlr24 — APPROVED** [review]
lgtm

**Review by inchori — APPROVED** [review]
LGTM

**Pull request merged**